python/equistore-operations/equistore/operations/join.py

```diff
@@ -17,9 +18,42 @@ def join(tensors: List[TensorMap], axis: str):
     `properties` dimension and for ``axis='samples'`` they will be the along the
     samples dimension.
 
-    ``join`` will create an additional label `tensor` specifiying the original index in
-    the list of `tensor_maps`.  If `sample`/`property` names are not the same in all
-    `tensor_maps` they will be unified with a general name ``"property"``.
+    Possible clashes of the meta data like samples/properties names are resolved by one
+    of the three following strategies:
+
+    1. If Labels names are the same and the values are unique, we keep the names and
+       join the values::
```

Comment:
why two double colons at the end? is this related to sphinx?

Reply:
Yes this renders the next paragraph as code.